packages/magic-entries-webpack-plugin/src/magic-entries-webpack-plugin.ts

```diff
@@ -3,37 +3,46 @@ import {basename, resolve} from 'path';
 import globToRegExp from 'glob-to-regexp';
 import {Compiler, Entry} from 'webpack';
 
+type EntryOption = Compiler['options']['entry'];
+
 export interface Options {
   pattern: string;
   folder: string | string[];
   nameFromFile: (file: string) => string;
+  appendToEntries: {[entryName: string]: string};
 }
 
-type EntryOption = Compiler['options']['entry'];
-
 /**
  * A webpack plugin that automatically configures webpack entries by checking the filesystem
  * @param config
```

Comment:
could we add docs here for what `appendToEntries` does? I wasn't really sure from just the name until I read through the code a few times